CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.1]
+
+### Fixed
+- Return correct contract id for `instantiate` command with subcontracts ‒ [#777](https://github.com/paritytech/cargo-contract/pull/777)
+
+### Changed
+- Bump template to ink! 4.0 ‒ [#971](https://github.com/paritytech/cargo-contract/pull/971)
```

Comment:
Put this in `Fixed` please since this was a mistake in the `2.0` release.

If it was a genuine `Change` this wouldn't be a patch release